tests/scalability/README.md

```diff
@@ -79,64 +79,16 @@ Github actions monitors the cht-core repo for a new beta tag. This triggers the
 
 The action checks out cht-core and then execute two scripts. One start medic-os related tasks and another to execute jmeter.
 
-### Start Medic Script
+### Start CHT Script
 
-The script `start_ec2_medic.sh` begins by requesting an ec2 instance in the ca-central-1 region. While it is starting it gets the public DNS name of that instace. The instance is started with `user-data` script located at `cht-core/tests/scalability/medic-os.sh`. AWS cli requires the script be encoded in base64. That step is handled in the start scripts.  This brings up medic-os in docker on that machine using the `prepare.sh` script in the `cht-infrastructure` repo. 
+The script `start-ec2-cht.sh` begins by requesting an ec2 instance in the ca-central-1 region. While it is starting it gets the public DNS name of that instance. The instance is started with `user-data` script located at `cht-core/tests/scalability/prepare-ec2.sh`. AWS cli requires the script be encoded in base64. That step is handled in the start scripts.  This brings up CHT 4.x in Docker on that machine.
 
-While docker compose is bringing up medic-os. `start_ec2_medic.sh` starts to check for API to be up by curling  `https://$PublicDnsName/api/info` until a version number is returned. 
+While docker-compose is bringing up CHT, `start-ec2-cht.sh` starts to check for API to be up by curling  `https://$PublicDnsName/api/info` until a version number is returned.
 
-Medic-conf and it's dependencies are now installed. Medic-conf then uploads standard config and the seed data which are located in `cht-core/config/standard` and `cht-core/tests/scalability/csv`. Then we wait for sentinel to process all these changes by checking the sentinel-meta-data processed sequence against the `_changes`  feed with the since request param. 
-
-From here we get the ddocs update sequences. Stage the branch which is associated to the tag that triggered this whole flow. Then check that the sequence numbers have increased. This is a simple sanity check that the views should have warmed. Then complete the upgrade. 
+Medic-conf and it's dependencies are now installed. Medic-conf then uploads the seed data which is located in `cht-core/tests/scalability/csv`. Then we wait for sentinel to process all these changes by checking the sentinel-meta-data processed sequence against the `_changes`  feed with the since request param.
```

Comment:
```suggestion
cht-conf and its dependencies are now installed. cht-conf then uploads the seed data which is located in `cht-core/tests/scalability/csv`. Then we wait for sentinel to process all these changes by checking the sentinel-meta-data processed sequence against the `_changes`  feed with the since request param.
```